src/ubidots.js

```diff
@@ -78,14 +87,34 @@ class Ubidots {
   /**
    * Create an instance of Ubidots API Client
    * @param {string} baseURL
+   * @param {object} opts
+   * @param {object} opts.backoff - backoff options
+   * @param {number=} opts.backoff.retries - amount of retries
+   * @param {number=} opts.backoff.minTimeout - minimum amount of time between retries
+   * @param {number=} opts.backoff.factor - factor of the timeout between retries
    * @returns {Ubidots}
    */
-  static create(baseURL = ApiBase.Industrial) {
+  static create(baseURL = ApiBase.Industrial, opts = { backoff: {}}) {
```

Comment:
Please define `options` in the following manner: `opts = Ubidots.DEFAULT_CONFIG`